pkg/commands/process/settings/rules/ruby/third_parties/bigquery.yml

```diff
@@ -3,15 +3,15 @@ patterns:
       $<INSERTER>.insert($<...>$<DATA_TYPE>$<...>)$<...>
     filters:
       - either:
-        - variable: INSERTER
-          detection: ruby_third_parties_bigquery_dataset
-        - variable: INSERTER
-          detection: ruby_third_parties_bigquery_insert_async
-        - variable: INSERTER
-          detection: ruby_third_parties_bigquery_table
+          - variable: INSERTER
```

Comment:
Unwanted change here?

Reply:
hmm. prettier may have fixed it. Only did it to this block and this file, so assuming there was some inconsistency. We might want to set up an editorconfig or equivalent at some point for the repo.